rss2email/email.py

```diff
@@ -146,8 +146,11 @@ def get_message(sender, recipient, subject, body, content_type,
         message.set_payload(body, charset=charset)
     if extra_headers:
         for key,value in extra_headers.items():
-            encoding = guess_encoding(value, encodings)
-            message[key] = _Header(value, encoding)
+            if key == 'Date':
+                message[key] = value
+            else:
+                encoding = guess_encoding(value, encodings)
+                message[key] = _Header(value, encoding)
```

Comment:
This would be redundant with [one of my changes](https://github.com/rss2email/rss2email/pull/166/commits/db959a8e1975211828806d16ac75513a37dcf9c6#diff-5858faa28f9fb55ca9746766095d02da651d8a85177f7b3d1c20d405a1d28c54R140) in #166.